llms/openai/openaillm_option.go

```diff
@@ -128,3 +130,10 @@ func WithResponseFormat(responseFormat ResponseFormat) Option {
 		opts.responseFormat = responseFormat
 	}
 }
+
+// WithLanguage allows setting a custom language.
```

Comment:
I'd expect this comment to hint at the format of these values and/or link to docs.